### ISSUE TITLE
Links need to be labelled consistently

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#110]
URL: https://hcdev.openplus.ca/en/alert-recall/test-uat-multiple-lots-irbesartan-losartan-and-valsartan-drugs-recalled-due-azido and https://hcdev.openplus.ca/en
User Agent: Firefox 93.0
Language: Both

Issue Description: Links “Report a health or safety concern” and “Submit report” (on the home page: https://hcdev.openplus.ca/en) go to the same page (https://www.canada.ca/en/services/health/report-health-safety-concern.html), but have different link text to describe them.
In addition, link text “Submit report” doesn’t seem to be appropriate as it sounds like a label for a button. In this case, link text on the home page should be changed to “Report a health or safety concern”

<a class="btn btn-danger" href="https://www.canada.ca/en/services/health/report-health-safety-concern.html" target="_blank">Submit a report</a>
![image](https://user-images.githubusercontent.com/68117281/137017938-35c07864-d949-4406-8bf4-c4efcc5a283f.png)

<a class="btn btn-default btn-lg" href="https://www.canada.ca/en/services/health/report-health-safety-concern.html">Report a health or safety concern</a>
![image](https://user-images.githubusercontent.com/68117281/137017923-9ab7cfe5-1927-44fb-aca2-00e46a11b05b.png)

Steps to reproduce: Navigate to both urls and visually scan the 'Report a health or safety concern' and 'Submit report' links.  Select each of them to navigate to new page.

Expected results: Components that have the same functionality across web application pages, should be identified consistently.

**Comment by ndolter** [comment]
Validated that the button has been corrected, however there appears to be a minor typo on the French button label:  'sécurité' is listed on one button and 'sécurité**n**' (https://hcdev.openplus.ca/fr/avis-rappel/test-uat-rappel-plusieurs-lots-medicaments-contenant-irbesartan-du-losartan-et-du) on the other (https://hcdev.openplus.ca/fr)

![image](https://user-images.githubusercontent.com/68117281/137193422-abf8df7b-8d47-41c4-9280-c2afa6515ead.png)

**Comment by echouin** [comment]
Sent back to OP

**Comment by echouin** [comment]
@ndolter OP confirms typ fixed.

**Comment by ndolter** [comment]
Validated that the typo has been corrected.
![image](https://user-images.githubusercontent.com/68117281/137383903-73c5ed8e-7473-45f6-aa81-522b0035ffcf.png)
